src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,26 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public static final int MAX_SIZE = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> charactersSet = new TreeSet<>();
+        try (FileReader fileReader = new FileReader(fileName)) {
+            int letter = fileReader.read();
+            while (letter != -1) {
+                if (Character.isLetter(letter)) {
+                    charactersSet.add((char)Character.toLowerCase(letter));
+                }
+                letter = fileReader.read();
+            }
+        }
+        StringBuilder stringBuilder = new StringBuilder();
+        Iterator iterator = charactersSet.iterator();
+        int size = 0;
+        while (iterator.hasNext() && size < MAX_SIZE) {
+            stringBuilder.append(iterator.next());
```

Comment:
We can use foreach loop and append char from foreach and then we   but not sure is that better or no. 
But great job!

Reply:
@andriylagun this is ok as well :)